docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[network-beaconing-framework]]
+== Network Beaconing
+
+This feature provides an early warning system for command and control beaconing activity. It monitors network traffic for indicators of compromise and provides analytics to add context to alerts and aid your threat hunting.
+
+[discrete]
+=== Deploy the package
+
+To deploy the network beaconing framework in your environment using the https://github.com/elastic/detection-rules/releases/tag/ML-Beaconing-20211216-1[release package], follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+
+The release package includes dashboards for monitoring beaconing activity in your environment. You can review signals via a Lens dashboard called Network beaconing.
+
+[role="screenshot"]
+image::images/beaconing-detection-1.png[]
+
+[discrete]
+=== Feature details
+
+This feature uses a {ref}/transforms.html[transform] to categorize network data by host and process name, then runs scripted metric aggregations on the host-process name pairs. For a given time window, the scripted metric aggregation checks each pair for the following:
+
+* Signals repeating at regular intervals, accounting for minor variations in those intervals.
```

Comment:
```suggestion
* Signals repeating at regular intervals, accounting for minor variations in those intervals
```

Reply:
I feel like I've looked for guidelines on this and haven't found anything, and I've probably seen it both ways in Elastic docs. IMO periods are OK for longer phrases like these.

Reply:
Yes, to what Joe said ^.